rats-processors/src/python/rats/pipelines_app/_decorators.py

```diff
@@ -0,0 +1,61 @@
+from collections.abc import Callable
+from typing import Concatenate, ParamSpec, TypeVar
+
+from rats import apps
+from rats.processors import ux
+
+T_ServiceType = TypeVar("T_ServiceType")
+P = ParamSpec("P")
+T_Container = TypeVar("T_Container", bound=apps.Container)
+
+
+def _process_method_to_task_method(
+    process_method: Callable[Concatenate[T_Container, P], ux.T_Processor_Output],
```

Comment:
This is a utility function used by the `task` decorator, so to be used on unbound methods.
I want to keep calling it `process_method` to make a clear distinction with the `task_method` that the function returns.  This is in line with our concept of task as wrappers over processors.

Reply:
I would try not to bind ourselves too much on the process method name. It can be something else, and we should eventually be flexible about it on classes. This converts any method into a Task, not a task method. So I would call it `method_to_task` or `function_to_task`. I think it is clearer and it describes what it does. Your call nonetheless.